### PR TITLE
feat: Rebuild exposedInMainWorld.d.ts in watch mode

### PR DESCRIPTION
**Pull request by dohooo**

### CODE REVIEW

**Comment by dohooo** [comment]
This will make the development process more enjoyable

File: scripts/watch.js [hunk 1]
```diff
@@ -100,6 +101,12 @@ const setupPreloadPackageWatcher = ({ws}) =>
     name: 'reload-page-on-preload-package-change',
     configFile: 'packages/preload/vite.config.js',
     writeBundle() {
+      // Generating exposedInMainWorld.d.ts when preload package is changed.
+      generate({
```
Comment (cawa-93) [inline scripts/watch.js:105]: Maybe you should use `generateAsync` here

Comment (dohooo) [reply scripts/watch.js]: Good idea！🍺

**Comment by cawa-93** [comment]
It would be great if you could fix the linter warnings: `Strings must use singlequote`

**Comment by dohooo** [comment]
> It would be great if you could fix the linter warnings: `Strings must use singlequote`

fixed 🍺！

**Pull request merged**